.gitignore

```diff
@@ -3,3 +3,4 @@ coverage/
 build/
 .eslintcache
 npm-debug.log
+.env
```

Comment:
I think this should have a new line, maybe missing `editorconfig` plugin or not set up properly?